modules/psap-driver-toolkit-using.adoc

```diff
@@ -0,0 +1,207 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-driver-toolkit.adoc
+
+[id="using-the-driver-toolkit"]
+= Using the Driver Toolkit
+
+As an example, the Driver Toolkit can be used as the base image for building a very simple kernel module called link:https://github.com/openshift-psap/simple-kmod[simple-kmod].
+
+[id="create-simple-kmod-image"]
+== Build and run the simple-kmod driver container on a cluster
+
+.Prerequisites
+
+* An {product-title} cluster
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+First create a namespace for the example:
+[source,terminal]
+-----
+$ oc new-project simple-kmod-demo
+-----
+
+. The following YAML defines an `ImageStream` for storing the simple-kmod driver container image, and a `BuildConfig` for building the container. Save this YAML as `0000-buildconfig.yaml.template`.
++
+[source,yaml]
```

Comment:
I understand this is an example but still will be a good idea to follow Dockerfile best practices (unless that does not really work) - 1) single RUN command and chain individual commands using && 2) perform cleanup (remove yum cache, yum clean all etc)

Reply:
Good point, I was able to combine several run commands and clear the dnf cache for the dnf installs.